src/content/reference/management-api/drainerconfigs.core.giantswarm.io.md

```diff
@@ -1,357 +0,0 @@
----
```

Comment:
Is this deletion (and others) ok?

Reply:
Yes, these were intentionally excluded (drainerconfigs and storageconfigs are deprecated, etcdbackups are internal)

Reply:
For reference: https://github.com/giantswarm/giantswarm/issues/15025

Reply:
Although to not break external links, we should consider keeping the pages around, clearly marking them as deprecated but not link them from the list.